examples/spellchecker/bindings/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    windows::build!(
+        windows::win32::intl::{ISpellChecker, SpellCheckerFactory, ISpellCheckerFactory},
+        windows::win32::com::*
```

Comment:
This generates a lot of code that is probably unused. Perhaps limit it to just the COM declarations you need.

Reply:
Oh yes, this helped a lot!